shared/actions/chat.js

```diff
@@ -536,40 +541,54 @@ function * _editMessage (action: EditMessage): SagaGenerator<any, any> {
   })
 }
 
-// Actually start a new conversation
-function * _startNewConversation (conversationIDKey: ConversationIDKey) {
-  const oldConversationIDKey = conversationIDKey
+// Actually start a new conversation. conversationIDKey can be a pending one or a replacement
+function * _startNewConversation (oldConversationIDKey: ConversationIDKey) {
   // Find the participants
-  const tlfName = pendingConversationIDKeyToTlfName(conversationIDKey)
-  if (tlfName) {
-    const users = tlfName.split(',')
-    const result = yield call(localNewConversationLocalRpcPromise, {
-      param: {
-        identifyBehavior: TlfKeysTLFIdentifyBehavior.chatGui,
-        tlfName: users.join(','),
-        tlfVisibility: CommonTLFVisibility.private,
-        topicType: CommonTopicType.chat,
-      }})
-    const newConversationIDKey = result ? conversationIDToKey(result.conv.info.id) : null
-    if (!newConversationIDKey) {
-      console.warn('No convoid from newConvoRPC')
-      return null
+  const pendingTlfName = pendingConversationIDKeyToTlfName(oldConversationIDKey)
```

Comment:
🐑 🐑 This could be a separate function: `oldConvId  -> tlfName` with the null check below